src/components/Menus/SideMenu/MenuSideBar.js

```diff
@@ -0,0 +1,31 @@
+import React from "react"
+import { useTranslation } from "react-i18next"
+import { DoubleChevronRight } from "../../../Icons"
+import Button from "../../Button/Button"
+import MenuSideBarOption from "./MenuSideBarOption"
+
+
+const MenuSideBar = (props) => {
+  const {menuOptions, option, toggleOption} = props
+  const { t } = useTranslation()
+  const topMenuOptions = menuOptions.filter((menuOption => menuOption.position === "top"))
+  const bottomMenuOptions = menuOptions.filter((menuOption => menuOption.position === "bottom"))
+
+  return (
+    <div className="menu-sidebar">
+      <div className={`menu-options-top`}>
+        {topMenuOptions.map((menuOption, i) => (
+            <MenuSideBarOption key={i} Icon={menuOption.icon} title={menuOption.title} isActive={option === menuOption.name} toggleOption={toggleOption} name={menuOption.name}/>
+          ))}
+      </div>
+      <div className={`menu-options-bottom`}>
+        {bottomMenuOptions.map((menuOption, i) => (
+            <MenuSideBarOption key={i} Icon={menuOption.icon} title={menuOption.title} isActive={option === menuOption.name} toggleOption={toggleOption} name={menuOption.name}/>
+          ))}
```

Comment:
Part of me feels this could be refactored a bit as it's basically the same thing. Let's leave it the ol' rule of three for now!